affirm_example/__init__.py

```diff
@@ -16,17 +16,27 @@ def _get_extra_request_args():
         return {}
 
 
-def get_checkout_from_token(checkout_token):
-    read_checkout_url = "{0}/checkout/{1}".format(app.config["AFFIRM"]["API_URL"], checkout_token)
+def _get_secret_api_key(public_api_key):
+    if "MERCHANTS" not in app.config["AFFIRM"]:
+        return app.config["AFFIRM"]["SECRET_API_KEY"]
+
+    for merchant in app.config["AFFIRM"]["MERCHANTS"]:
```

Comment:
Can you add a comment to indicate that the two approaches are mutually exclusive.  Either that or merge the default merchant with the ones in "MERCHANTS"

Reply:
added a comment explaining that if `PUBLIC_API_KEY` and `SECRET_API_KEY` are also set (in addition to `MERCHANTS`), they will serve as the default selected merchant